angular-offline.js

```diff
@@ -79,15 +102,27 @@ angular
     }
 
     /**
+     * Return cache/default cache
+     *
+     * @param {object} cache Cache
+     * @returns {object} Cache
+     */
+
+    function getCache (cache) {
+      if (cache === true)
```

Comment:
Well it was forked from an external lib which is no longer maintained, so it doesn't have our style rules applied.